logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -180,3 +190,31 @@ data class MemberDetails(val user: User, val role: Conversation.Member.Role)
 typealias ClientId = PlainId
 
 data class Recipient(val id: UserId, val clients: List<ClientId>)
+
+data class ConversationView(
```

Comment:
Same here.
It could be `ConversationDetails`.

And we can throw the current `ConversationDetails` in the bin.

Reply:
But why do we have to expose this flat data class? The old ConversationDetails is much nicer.

Reply:
I don't see why can't ditch `ConversationView` and instead map to the existing `ConversationDetails`.